### PR TITLE
Update ios.md

### PR DESCRIPTION
**Pull request by bryanjclark**
Clarifying that "interactors" aren't responsible for user-interaction gestures, but rather interacting with side effects.

### CODE REVIEW

**Comment by andymatuschak** [comment]
Thanks!

File: style/ios.md [hunk 1]
```diff
@@ -69,11 +69,11 @@ Interactions with these frameworks embody a complex responsibility; their vended
 
 `UIViewController`s often entangle many responsibilities:
 
-1. Receiving lifecycle and system events from UIKit and instantiating `UIView`s.
+1. Receiving lifecycle and system events from UIKit and instantiating `UIView`s, and handling user-interaction gestures.
```
Comment (NachoSoto) [inline style/ios.md]: Now there are one too many "and"s.

**Comment by NachoSoto** [comment]
LGTM!

File: style/ios.md (hunk 1 above)
Comment (andymatuschak) [inline style/ios.md]: Fixed.

**Pull request merged**